src/services/funcService.test.ts

```diff
@@ -1,10 +1,11 @@
-import fs from "fs";
 import mockFs from "mock-fs";
 import rimraf from "rimraf";
 import { MockFactory } from "../test/mockFactory";
 import Serverless from "serverless";
 import { FuncService } from "./funcService";
 
+import fs from "fs";
```

Comment:
nit: weird reformat that move this line and added blank line from previous section